### ISSUE TITLE
Field limits report

### ISSUE THREAD

**Issue by SayChi** [engram-design/FieldManager#63]
It would be awesome to see an overview that shows how fields are being filled in. Think of asset select fields that are only ever being used with 1 image while no limit is set on the field. It would be great to have an overview of such things so future expansions of a site can take that into account.
It would also be usefull if you just forgot to put limits on fields and you can backtrack to fix those with such an overview.

Maybe the audit tab is a good place for this.

**Comment by engram-design** [comment]
Yeah, interesting. I suppose it just depends on how far we want to run with this idea - not just limits, but other things related to field options. I'll have a think about how best to show this.
